scripts/get_config_map_status.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+CONFIGMAP_NAME="iks-ca-configmap"
+NAMESPACE="kube-system"
+COUNTER=0
+MAX_ATTEMPTS=40
+
+while [[ $COUNTER -lt $MAX_ATTEMPTS ]] && ! kubectl get configmap $CONFIGMAP_NAME -n $NAMESPACE &>/dev/null; do
+  COUNTER=$((COUNTER + 1))
+  echo "Attempt $COUNTER: ConfigMap '$CONFIGMAP_NAME' not found in namespace '$NAMESPACE', retrying..."
+  sleep 60
+done
+
+if kubectl get configmap $CONFIGMAP_NAME -n $NAMESPACE &>/dev/null; then
```

Comment:
instead of running the command again here, why not just use a boolean variable inside the while loop?